AutoGVP/01-annotate_variants_CAVATICA_input.R

```diff
@@ -318,11 +318,12 @@ combined_tab_with_vcf_intervar <- autopvs1_results %>%
     evidencePVS1 = if_else((criterion == "IC4") & evidencePVS1 == 1, "0", evidencePVS1),
 
     # if criterion is na|NF2|NF4|SS2|SS4|SS7|DEL3|DEL5|DEL9|DUP2|DUP4|DUP5|IC5 then PVS1 = 0;
-    evidencePVS1 = if_else((criterion == "na" | criterion == "NF2" | criterion == "NF4" |
-      criterion == "SS2" | criterion == "SS4" | criterion == "SS7" |
-      criterion == "DEL3" | criterion == "DEL5" | criterion == "DEL9" |
-      criterion == "DUP2" | criterion == "DUP4" | criterion == "DUP5" |
-      criterion == "IC5") & evidencePVS1 == 1, 0, as.double(evidencePVS1)),
+    evidencePVS1 = if_else((criterion == "na" | criterion == "NF0"|criterion == "NF2"| criterion == "NF4" |
+      criterion == "SS2" | criterion == "SS4"|criterion == "SS7"|
+      criterion == "DEL3" | criterion == "DEL5"|criterion == "DEL9"|
+      criterion == "DUP2" | criterion == "DUP4"|criterion == "DUP5"|
+      criterion == "IC5" ) & evidencePVS1 == 1, 0, as.double(evidencePVS1)),
+
```

Comment:
```suggestion
    evidencePVS1 = if_else((criterion == "na" | criterion == "NF0" | criterion == "NF2" | criterion == "NF4" |
      criterion == "SS2" | criterion == "SS4" | criterion == "SS7" |
      criterion == "DEL3" | criterion == "DEL5" | criterion == "DEL9" |
      criterion == "DUP2" | criterion == "DUP4" | criterion == "DUP5" |
      criterion == "IC5" ) & evidencePVS1 == 1, 0, as.double(evidencePVS1)),
```
adding spacing. Side note - @naqvia do we really have "na" in some file rather than `NA`?

Reply:
Yes, I usually see them as "na"